libs/rs-sdk-integration-tests/src/http.rs

```diff
@@ -63,3 +64,72 @@ pub fn test_http_post_success() {
 
     Process::error(&response.bytes);
 }
+
+/**
+ * Specify 1000ms timeout for an endpoint that responds in 5000ms.
```

Comment:
To make our lives easier maybe we can leave out the exact numbers and just specify that the timeout is shorter than the time the endpoint will take. Although if we change the test name that's implied and we can maybe get rid of the comment.